components/sup/src/manager.rs

```diff
@@ -994,7 +1000,12 @@ impl Manager {
             loop {
                 match *started {
                     http_gateway::ServerStartup::NotStarted => {
-                        started = match cvar.wait_timeout(started, Duration::from_secs(10)) {
+                        started = match cvar.wait_timeout(started,
+                                                          self.state
+                                                              .cfg
+                                                              .http_startup_timeout
+                                                              .into())
```

Comment:
Strictly speaking we do not need to add `http_startup_timeout` to `ManagerConfig`, correct? We could simply use `HttpStartupTimeout::configured_value()` here. I think it is fine to leave in `ManagerConfig` because it sets us up to easily add `http_startup_timeout` as a cli option in the future, but I thought I would bring it up as a potential simplification.

Reply:
OMG!!! You are so right. I think when I started this I did not know that the `env_config` just creates a type and was not aware it would be accesible to manager.